routes/api/users.js

```diff
@@ -81,10 +95,51 @@ router.get("/current", authMiddleware, async (req, res, next) => {
         return res.status(200).json({
             email: currentUser.email,
             subscription: currentUser.subscription,
+            avatarURL: newUser.avatarURL,
         });
     } catch (err) {
         next(err);
     }
 });
 
+router.post(
+    "/avatars",
+    authMiddleware,
+    uploadMiddleware.single("avatar"),
+    async (req, res, next) => {
+        if (!req.user) {
+            return res.status(401).json({ message: "Not authorized" });
+        }
+        if (!req.file) {
+            return res.status(400).json({ message: "No file uploaded" });
+        }
+        const { path: temporaryPath } = req.file;
+        const extension = path.extname(temporaryPath);
+        const fileName = `${uuidV4()}${extension}`;
```

Comment:
Zastanawiam jak to się stało, że tu masz taką nazwę pliku a obrazek który przesłałaś nosi nazwę `test-image.png`?